### PR TITLE
EUI-5583 View details waiting to be listed for JOH

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5583
https://tools.hmcts.net/jira/browse/EUI-5584
https://tools.hmcts.net/jira/browse/EUI-5585

### Change description ###
View details for JOH

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: api/hearings/data/hearingLists.mock.data.ts [hunk 1]
```diff
@@ -316,11 +316,37 @@ const HEARING_DAY_SCHEDULE_13: HearingDayScheduleModel = {
...
+  hearingRoomId: 'room 5',
+  hearingJudgeId: 'hearingJudgeId1',
+  panelMemberIds: ['hearingJudgeId1'],
+  attendees: [
+    {
+      partyID: 'P1',
+      partyName: 'Jane and Smith',
+      partyType: PartyType.IND,
+      partyRole: 'claimant',
+      partyChannel: 'inPerson',
```
Comment (kuda1992) [inline api/hearings/data/hearingLists.mock.data.ts:333]: `partyChannel` has been removed from the `PartiesModel` refer to https://github.com/hmcts/rpx-xui-webapp/pull/1919/files

File: api/hearings/models/hearings.enum.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export enum Actions {
 }
 
 export enum HMCStatus {
-  HEARING_REQUESTD = 'Hearing requested',
+  HEARING_REQUESTED = 'Hearing requested',
```
Comment (rayliangatos) [inline api/hearings/models/hearings.enum.ts:9]: thanks for spot this, could you also amend the one in src/../hearings.enum.ts

**Review by kuda1992 — CHANGES_REQUESTED** [review]
I will wait until my pr goes in, it has model changes you might need to incorporate, otherwise looks good

File: api/hearings/models/hearings.enum.ts (hunk 2 above)
Comment (kuda1992) [reply api/hearings/models/hearings.enum.ts:9]: 😭 I was wondering what changed. So easy to miss.

File: api/hearings/data/hearingLists.mock.data.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply api/hearings/data/hearingLists.mock.data.ts:333]: Sure @kuda1992 👍

File: api/hearings/models/hearings.enum.ts (hunk 2 above)
Comment (johnbenjamin-hmcts) [reply api/hearings/models/hearings.enum.ts:9]: Thank you @rayliangatos I have now updated the code.

**Review by kuda1992 — APPROVED** [review]
LGTM. Thanks for making the changes @johnbenjamin-hmcts.

**Review by rayliangatos — APPROVED** [review]

**Pull request merged**